README.md

```diff
@@ -0,0 +1,15 @@
+Installation Instructions
+=========================
+For Installation Instructions see http://www.dokuwiki.org/install
+
+Dokumentation
+=============
+All documentation for DokuWiki is available online at http://www.dokuwiki.org/.
+
+Copyright & License
+===================
+DokuWiki - 2004-2019 (c) Andreas Gohr <andi@splitbrain.org> 
+                         and the DokuWiki Community 
+
+See COPYING and file headers for license info
+
```

Comment:
I am not sure if we want to convert this to a markdown file.

Reply:
One could argue that README.md is what one would expect, but ... looking from a users perspective, maybe not that into tech, a .txt file is better here. Has been the last 15 years this way, not that this would be a meaningful argument ;)